recipes-sota/config/files/60-test-polling-interval.toml

```diff
@@ -0,0 +1,2 @@
+[uptane]
+polling_sec = 30
```

Comment:
Wouldn't we want the actual value to be parametrizable, like here https://github.com/advancedtelematic/meta-updater/blob/master/recipes-sota/aktualizr/files/10-resource-control.conf?

Reply:
Not a bad idea. Maybe 30 should be the default and then have a variable that the user can control?

Reply:
Yes, that's basically what I had in mind, we have the same mechanism there https://github.com/advancedtelematic/meta-updater/blob/550ef5c1c32be58cca91f80afbedc23d5e9b79b6/recipes-sota/aktualizr/aktualizr_git.bb#L65

Reply:
@lbonn Done, I think. What do you think now?